packages/playground/package.json

```diff
@@ -59,7 +59,7 @@
   "devDependencies": {
     "@types/mocha": "~9.1.0",
     "@types/node": "~16.0.3",
-    "@types/prettier": "^2.0.2",
+    "@types/prettier": "2.6.0",
```

Comment:
Would ~2.6.0 be OK?

Reply:
nah :( , the problem was introduced in 2.6.2

Reply:
this is just pinning it to the version we already were using